build-tools/package.json

```diff
@@ -125,16 +125,16 @@
 			"oclif>@aws-sdk/client-cloudfront": "npm:empty-npm-package@1.0.0",
 			"oclif>@aws-sdk/client-s3": "npm:empty-npm-package@1.0.0"
 		},
-		"patchedDependencies": {
-			"jssm@5.98.2": "patches/jssm@5.98.2.patch"
-		},
 		"updateConfig": {
 			"ignoreDependencies": [
 				"latest-version",
 				"read-pkg-up",
 				"type-fest",
 				"typescript"
 			]
+		},
+		"patchedDependencies": {
+			"jssm-viz@5.101.0": "patches/jssm-viz@5.101.0.patch"
```

Comment:
Thoughts on leaving a comment here saying that we will remove this once jssm-viz fixes the issue? Could potentially also link to an ADO item as well if we are worried about this getting lost

Reply:
Done in latest.